pkg/pop/types.go

```diff
@@ -0,0 +1,25 @@
+package pop
+
+import (
+	"github.com/Azure/kubelogin/pkg/internal/pop"
+)
+
+type PoPAuthenticationScheme struct {
```

Comment:
add a comment on top of this that describe this is the msal implementation of AuthenticationScheme (You should add a link to msal repo interface)